### ISSUE TITLE
Plugin doesn't seem to use .golangci.yml exclude-rules

### ISSUE THREAD

**Issue by chrisnappin** [xxpxxxxp/intellij-plugin-golangci-lint#126]
Hi, we're using the plugin on our current work project (thanks to the recent plugin fix on honouring no modules mode), but we've noticed that disabled linting rules are being reported on our unit test code.

We have the following config in the .golangci.yml files in the top level directory of each of our repos, which disables certain rules for unit test code only:

```
issues:
  exclude-rules:
    # lint rules disabled in unit test code
    - path: '(.+)_test\.go'
      linters:
        - forcetypeassert
        - goerr113
        - varnamelen
        - wrapcheck
```

However when opening a unit test source code file, we still see linting rules being reported (e.g. the "forcetypeassert" rule).

When running golangci-lint from the command line with the same config we don't see these linting rules being run on our unit test code.

Please note that for the path I've tried `_test\.go` and `(.+)_test\.go`, either work at the command line, but neither seems to have any impact on the plugin

**Comment by chrisnappin** [comment]
Sure, screenshot attached.

The relative file structure is as follows:
* project top level directory
    * .golangci.yml
    * sub directories
        * unit test files  

for example, `worker_pool/worker_pool_test.go` or `kafka/publisher_test.go`

**Comment by xxpxxxxp** [comment]
Hi @chrisnappin , I tried to reproduce the issue with https://github.com/xxpxxxxp/debug
The exclude-rule worked. Please try with your IDE, do quit and restart your IDE after changing` .golangci.yml`, cause there's in memory cache. If the issue still occur, I need your plugin settings for debug then.

**Comment by chrisnappin** [comment]
Thanks for investigating, I wasn't aware the entire IDE needed restarting each time the config file is changed, although I thought I had tried that. It seems to be working if I do that, so closing this issue as a non-issue.
